### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by leoujz** [OpenKNX/GW-REG1-Dali#1]
```
src/DaliChannel.cpp:663:41: error: 'SET_TEMP_KELVIN' is not a member of 'DaliCmdExtendedDT8'
src/DaliModule.cpp:1406:51: error: 'SET_TEMP_KELVIN' is not a member of 'DaliCmdExtendedDT8'
src/DaliModule.cpp:1455:69: error: 'QUERY_COLOR_VALUE' is not a member of 'DaliCmdExtendedDT8'
src/DaliModule.cpp:1459:61: error: 'QUERY_COLOR_VALUE' is not a member of 'DaliCmdExtendedDT8'
src/DaliModule.cpp:1463:61: error: 'QUERY_COLOR_VALUE' is not a member of 'DaliCmdExtendedDT8'
```
Encountered these errors during building, I use the master branch's latest commit, and put the dependencies into the lib folder. I checkout to specific branch/tag and specific commit for each dependency described in the dependencies.txt
```
------- Built with -------
9d7c8ac v1dev knx https://github.com/OpenKNX/knx.git
0ba2086 v1 OFM-FileTransferModule https://github.com/OpenKNX/OFM-FileTransferModule.git
84a1e7b v1 OGM-Common https://github.com/OpenKNX/OGM-Common.git
a00d543 main arduino-dali https://github.com/thewhobox/arduino-dali
```
After checking the arduino-dali library, I found the code
```
enum DaliCmdExtendedDT8 {
  SET_TEMP_KELVIN = 231,
  SET_TEMP_RGB = 235,
  QUERY_COLOR_TYPE_FEATURES = 249
};
```
it does not contain SET_TEMP_KELVIN and QUERY_COLOR_VALUE, so it might be some thing mismatched, thanks for giving some hints or instructions.

**Comment by thewhobox** [comment]
Hey,

ist should now work.
Please pull the newest commit from GW-REG1-Dali and restore the dependencies with thee restore script or via task in vscode.

**Comment by leoujz** [comment]
Thanks, finally build it successfully!

Pulling the latest commit(85b0685), and update the dependencies as described in dependencies.txt, all develop tasks were built successfully, except for the release task. 

It reports `ParamBASE_Watchdog is not declared` in `lib/OGM-Common/src/OpenKNX/Common.cpp`, I add a line `#define ParamBASE_Watchdog 1` in file `include/knxprod.h`, then I successfully built the release task. I noticed that knxprod.h is generated by [Kaenx-Creator](https://github.com/OpenKNX/Kaenx-Creator) or [OpenKNXproducer](https://github.com/OpenKNX/OpenKNXproducer), it might be something missing or mismatched.

By the way, we are testing the DALI interface mentioned in tag: [REG1-App-Dali_V00.90](https://github.com/OpenKNX/GW-REG1-Dali/releases/tag/REG1-App-Dali_V00.90), I noticed that hardware files are removed in the latest commit of the main branch, is there any probem with the hardware files, or just moved to other places, or to be not public anymore? thanks!
